test/extended/authentication/keycloak_client.go

```diff
@@ -104,7 +104,7 @@ func (kc *keycloakClient) CreateUser(username, password string, groups ...string
 		Groups:        groups,
 		Credentials: []credential{
 			{
-				Temporary: true,
+				Temporary: false,
```

Comment:
Testing locally, this looked like it was causing issues where the users we created were considered to be "incomplete" because they needed to set a real password.